docs/advanced_guide.rst

```diff
@@ -0,0 +1,27 @@
+.. _advanced_guide:
+.. :orphan:
```

Comment:
Regarding the warnings, getting file missing from toctree warnings that's supposed to be fixed by adding these orphan tags but it doesnt seem to be working. @8bitmp3 any idea how I can get around this?

<img width="1790" alt="image" src="https://user-images.githubusercontent.com/7213793/206567140-adfb627c-aac5-4b8d-a398-f049a8a58c6e.png">

Reply:
ill try it1 I knew there had to be some magic parameter!